src/main.ts

```diff
@@ -76,8 +67,8 @@ function loadSettingsFromArgs(args: string[]): {settings: Settings, tscArgs: str
         process.exit(1);
     }
   }
-  if (!settings.outputPath && !settings.externsPath) {
-    console.error('must specify --output or --externs path');
+  if (!settings.externsPath) {
```

Comment:
Because this is required, it might be worth making this an argument instead of an option. Alternatively, do not make it required.